unsuccessful_login_spec.js

```diff
@@ -14,7 +15,7 @@ describe('Protractor Demo App Login', function () {
         await loginPage.login("viktoriasynytsa@gmail.com", "");
         expect(await loginPage.getErrorMessage()).toEqual('Password is required.');
         //await browser.sleep(5000);
-        await loginPage.clearCredentials();
+        await utils.clearCredentials();
```

Comment:
`clearCredentials ` should be part of the login page, not utils

Reply:
fixed